src/commandHandlers/issue.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import { load as cheerioLoad } from 'cheerio';
+import { Message, MessageEmbed } from 'discord.js';
+
+/**
+ * This command lets the user create an issue from discord to a github repository
+ */
+export const command = async (arg: string, embed: MessageEmbed, message: Message) => {
+    message.channel.send('To which repository would you like to create an issue?')
+    try {
+        // await repository name
+        const repository = await message.channel.awaitMessages((m: Message) => m.author.id === message.author.id, { max: 1, time: 10000, errors: ['time'] }).then(collected => {
+            const res = collected.first();
+            if(res) return res.content;
+            else return 'EddieBot';
+        }).catch(() => { throw new Error('You took too long to reply (`10 seconds`). Try again.') });
+
+        let url = 'https://github.com';
+        const { data } = await axios.get(`https://github.com/search?q=${encodeURI(repository)}`);
+        const $ = cheerioLoad(data);
+        const element = $('a.v-align-middle').first();
+        const attr = element.attr();
```

Comment:
does this implementation have an impact on performance? I'm just asking since I don't know what cheerio does and what is the cost of scraping GitHub's website instead of calling their API.

Reply:
I didn't find a search endpoint on their API. If there is one, it should have better performance.
I can't comment much on cheerios performance as I don't know much about its internals.

Reply:
@itsphaze [Search api](https://docs.github.com/en/free-pro-team@latest/rest/reference/search)

Reply:
I'm so blind :(
I'll implement this with the API